Added SA functionality

Strandgaard96 opened the pull request:
Added folder with code to calculate SA scores.

Strandgaard96 committed:
Added SA functionality

tlinjordet reviewed (changes requested):
I tried to run code but it is missing at least a utility `.py` file. 

This line fails for running both `neutralize.py` and `sascorer.py` as main.
```
from utils.gaussians import gaussian_modifier_clipped
```

To add this functionality now (and any other in the future) we should follow the template-based workflow format as for all the other process steps, including the bash files, the formatting style, the documentation standard, having clearly defined endpoints (i.e., datafiles) to compare against. This should also include a consistent extension of the DETAILS.md files on both workflow and datasets. 

If possible, it would be good to have this before we put the v1.0 tag on the public repository.

Strandgaard96 committed:
Added missing gaussians and simplified run example

Strandgaard96 committed:
changed readme slightly

Strandgaard96 commented:
Great yeah the gaussians should be there. Added the missing functions and simplified the run example :) 

There really should be no need to alter any of this. Creating a workflow on top of this would be unnecessary bloat. The calculateScore function was simply used on a list of SMILES to create the histograms in the preprint. 

The calculateScore function to get the SA would be the only function anyone would be interested in.

Strandgaard96 committed:
Update docs slightly
